FreeRTOS/Demo/Posix_GCC/main_networking.c

```diff
@@ -47,37 +47,37 @@
 /*#include "SimpleUDPClientAndServer.h" */
 /*#include "SimpleTCPEchoServer.h" */
 /*#include "TCPEchoClient_SingleTasks.h" */
-/*#include "demo_logging.h" */
+/*#include "logging.h" */
```

Comment:
The only real change here is `demo_logging.h` -> `logging.h`. The rest of the changes are from uncrustify.